include/swift/AST/ExperimentalDependencies.h

```diff
@@ -939,8 +939,9 @@ template <typename GraphT> class DotFileEmitter {
 // MARK: Declarations for YAMLTraits for reading/writing of SourceFileDepGraph
 //==============================================================================
 
-// This introduces a redefinition for Linux.
-#if !defined(__linux__)
+// This introduces a redefinition where ever std::is_same_t<size_t, uint64_t>
+// holds
+#if !(defined(__linux__) || defined(_WIN64))
 LLVM_YAML_DECLARE_SCALAR_TRAITS(size_t, QuotingType::None)
```

Comment:
I question whether this should be here at all—it's polluting the general namespace by providing a definition for everyone for a type that this header doesn't own (`size_t`). YAMLTraits.h doesn't declare a ScalarTraits for any non-fixed-sized types, so it actually seems deliberate. Maybe we should be using `int32_t` anyway?

(Sorry for not getting back to your full PR. Merging without review is okay because it's still opt-in, but I do think there's still more review to be done, whether from me or someone else.)

Reply:
@jrose-apple - was that meant for @davidungar?  I'm simply trying to restore the Windows builds :-p.  If there is something else that needs to be done, depending on the complexity, I might be able to pitch in.

Reply:
Yeah, it's meant for David, sorry. I think it's okay to put this in while waiting for his response.